src/scripts.js

```diff
@@ -65,25 +194,25 @@ let sortedHydrationDataByDate = user.ouncesRecord.sort((a, b) => {
 });
 
 for (var i = 0; i < dailyOz.length; i++) {
-  dailyOz[i].innerText = user.addDailyOunces(Object.keys(sortedHydrationDataByDate[i])[0])
+  dailyOz[i].innerText = currentUser.addDailyOunces(Object.keys(sortedHydrationDataByDate[i])[0])
 }
 
 hydrationUserOuncesToday.innerText = hydrationData.find(hydration => {
-  return hydration.userID === user.id && hydration.date === todayDate;
+  return hydration.userID === currentUser.id && hydration.date === todayDate;
 }).numOunces;
 
 hydrationFriendOuncesToday.innerText = userRepository.calculateAverageDailyWater(todayDate);
 
 hydrationInfoGlassesToday.innerText = hydrationData.find(hydration => {
-  return hydration.userID === user.id && hydration.date === todayDate;
+  return hydration.userID === currentUser.id && hydration.date === todayDate;
 }).numOunces / 8;
 
 
 function waterCardHandler() {
-  let hydrationCalendarCard = document.querySelector('#hydration-calendar-card');
-  let hydrationFriendsCard = document.querySelector('#hydration-friends-card');
-  let hydrationInfoCard = document.querySelector('#hydration-info-card');
```

Comment:
Hadn't realized we needed to put those variables inside the water handler, sorry about that!